_posts/2015-10-30-blend.md

```diff
@@ -0,0 +1,220 @@
+en invitant davantage d’intervenants d’autres pays, mais aussi en mixant encore plus les compétences des différents
+speakers. Pourquoi ne pas proposer un nouveau format de conférence, animé par deux personnalités aux cœurs de métiers
+différents ? En tous les cas, j’espère être là l’an prochain, et que Les Bricodeurs gagnent un autre Troll d’Or !
+
+---
+
+<img src="/images/avatars/quentinfayet.jpeg" class="bio-photo" alt="Quentin Fayet bio photo" style="max-width: 150px; border-radius: 150px; float: left; margin-right: 20px;">
+
+##Retour sur le Blend par Quentin Fayet
+
+#### ([@LimonPies](https://twitter.com/LimonPies){: target="_blank"})
+
+<br/>
+
+Pour ma première fois au Blend Web Mix, j’en ai pris plein les yeux et plein les oreilles. Entre des conférences de
+qualité menées par des orateurs experts dans leurs domaines et non avares en gifs, des hackathons divers et variés avec
+des équipes sur-vitaminées, le Blend m’a ouvert sur une communauté web lyonnaise - et plus largement française -
+incroyablement dynamique, ouverte, et à la pointe de la technologie.
+
+##### Conférence sur 5 fausses idées reçues sur la Silicon Valley par Anji Ismail
+([@anjismail](https://twitter.com/anjismail){: target="_blank"})
+
+Anji Ismaïl, CEO de [Doz](http://www.doz.com){: target="_blank"}, est venu partager avec nous son retour d’expérience sur l’implantation de Doz
+dans la Silicon Valley, et, au passage, briser quelques mythes et on-dit à propos de ce lieu qui, depuis des années,
+fait fantasmer les techos et autres geeks.
+
+Le premier a priori qu’Anji met à l’épreuve est celui selon lequel il serait “plus facile de créer sa boîte dans la
+Silicon Valley” (qu'en france, ndlr). Il nous dévoile alors la difficulté d'appréhender l’administration américaine,
+particulièrement lourde pour un français, ainsi que la facilité avec laquelle les géants qui ont élu domicile dans la
+Silicon Valley enchaînent les procès. Un environnement stimulant, mais extrêmement périlleux pour une petite startup
+française, donc.
+
+Par la suite, Anji Ismail lève le voile sur la rumeur selon laquelle il serait “plus facile de recruter” dans la
+Silicon Valley. En nous introduisant les concepts de “Coding Academy” (des écoles de coding très rapides, dont les
+diplômés réclament des salaires très élevés), de la “Guerre des Talents” (le fait que les entreprises débauchent les
+candidats des unes les autres), et des “A players” (les “personnalités” techniques dans la place), il nous explique que
+recruter (et conserver) ses employés n’est en fait pas si simple. Qui plus est, des “steal startups”, sortes de
+startups secrètes, attirent et retiennent les tops talents avant même leur création officielle.
+
+Le mythe selon lequel “lever des fonds est plus facile” est “presque vrai”, nous dit Anji. Outre le fait que les fonds
+d’investissements ne retiennent qu’une trentaine de dossiers sur les milliers qu’ils reçoivent, il faut également se
```

Comment:
I hesitated for the `s`. But as you can notice few lines below, there is no `s` in "investissement" ^^